### DISCUSSION TITLE
能够更新下commandgroup和matchergroup的例子

### DISCUSSION THREAD

**Discussion by BigCat-byebye** [nonebot/discussions#424] [通用]
能够更新下commandgroup和matchergroup的例子，举个代码例子

**Comment by yanyongyu** [comment]
现有文档例子有什么问题吗？https://nonebot.dev/docs/next/advanced/matcher#%E5%93%8D%E5%BA%94%E5%99%A8%E7%BB%84

**Reply by BigCat-byebye** [reply]
代码如下
```python
from nonebot.adapters.telegram.bot import Bot
from nonebot import on_command, CommandGroup
from nonebot.adapters.telegram.message import Message
from nonebot.params import CommandArg

adminGroup = CommandGroup("admin")
cmd = adminGroup.command(tuple())
status = adminGroup.command("status")

@cmd.handle()
async def cmd_test():
    await cmd.finish("cmd test")

@status.handle()
async def check():
    await status.finish("status check")
```

结果如下
![image](https://github.com/nonebot/discussions/assets/32433624/7455c075-c15c-49f2-b14a-458a7b8eb856)


我不太理解为什么只会响应cmd这个，而不响应status。。。。

**Reply by BigCat-byebye** [reply]
即使我将cmd的这个响应器和下面对应的cmd的函数也注释了，再输入/admin.status ，也还是无法响应status

**Reply by yanyongyu** [reply]
看上去这个是telegram的slash command问题，因为不支持分隔符导致无法识别

@nonebot/telegram
